guides/New Mapper Mapmerge Readme.md

```diff
@@ -0,0 +1,7 @@
+If you're making a PR to Github, or are a new mapper and have been told to "mapmerge":
```

Comment:
Don't we have more automatic tools for mapmerge?

Reply:
IIRC we do, but this is an older readme intended for VOREStation's method of doing it

Reply:
ater; StrongDMM can output to TGM so yes, there are technically "automatic tools" in that regard. I assume other mapping tools like spacemanDMM can do it as well.

rykka; You might want to mention in the options setup that people should switch SDMM's output to TGM since you're already recommending them to use it and it'll save them arsing around with bat files. I believe it's the default setting nowadays but it doesn't hurt to have people check.